finite-state/shared/src/main/scala/fs2/data/mft/query/QueryCompiler.scala

```diff
@@ -58,7 +58,7 @@ private[fs2] abstract class QueryCompiler[Tag, Path] {
   /** Return the constructor tag of this pattern, or `None` if it is a wildcard. */
   def tagOf(pattern: Pattern): Option[Tag]
 
-  def compile(query: Query[Tag, Path]): MFT[NonEmptyList[Guard], Tag, Tag] = {
+  def compile(query: Query[Tag, Path], credit: Int = 50): MFT[NonEmptyList[Guard], Tag, Tag] = {
```

Comment:
I think this one could use some (scala)docs? Where does this magic value 50 come from, why would people want to pass something different…

Reply:
True.